dojo/templates/issue-trackers/jira_full/jira-description.tpl

```diff
@@ -40,7 +40,7 @@
 
 
 {% if finding.component_name %}
-Vulnerable Component: {{finding.component_name }} - {{ finding.component_version }}
+*Vulnerable Component*: {{finding.component_name }} - {{ finding.component_version }}
 
 {% endif %}
 {% if finding.sast_source_object %}
```

Comment:
Wouldn't it make the tags for the SAST fields bold as well?
```
*Source Object:* {{ finding.sast_source_object }}
*Source File:* {{ finding.sast_source_file_path }}
*Source Line:* {{ finding.sast_source_line }}
*Sink Object:* {{ finding.sast_sink_object }}
```

Reply:
Indeed @StefanFl , rebased